docker/vpp-vswitch/govpp.conf

```diff
@@ -0,0 +1,3 @@
+health-check-probe-interval: 1000000000
+health-check-reply-timeout: 500000000
+health-check-threshold: 1
```

Comment:
This needs a line at the end of the file. ;)

Reply:
dang, i thought i fixed it. ..

Reply:
done